src/vs/workbench/contrib/chat/browser/chatListRenderer.ts

```diff
@@ -732,6 +731,23 @@ export class ChatListItemRenderer extends Disposable implements ITreeRenderer<Ch
 		this.updateItemHeightOnRender(element, templateData);
 	}
 
+	private shouldShowWorkingProgress(element: IChatResponseViewModel, partsToRender: IChatRendererContent[]): boolean {
```

Comment:
This is different than what `shouldShowWorkingProgress` did before, is that right?

Reply:
My mistake. Pushing fix for this so it is aligned with how it was before.